target/agon/include/video_h.asm

```diff
@@ -0,0 +1,80 @@
+; SPDX-FileCopyrightText: 2023 Zeal 8-bit Computer <contact@zeal8bit.com>
```

Comment:
Do you use this file? You can get rid of it to simplify the port if you don't use it

Reply:
As it happens I do use a few of the defines in here for video control modes.